frontend/src/components/roadmap/sections/RoadmapMilestones.tsx

```diff
@@ -37,8 +34,8 @@ export default function RoadmapMilestones() {
           items={[
             {
               title: t("archivedRoadmapTitle"),
-              headingLevel: "h2",
-              className: "font-sans-md tablet:font-sans-lg",
+              headingLevel: "h3",
+              className: "",
```

Comment:
- Should be an `h3` (the section heading is an `h2`) 
- Don't need the classes (could probably remove the empty `className`)